src/components/include/cbuf.h

```diff
@@ -18,11 +18,12 @@
 #define  CBUF_H
 
 #include <cos_component.h>
+#include <util.h>
 #include <cos_debug.h>
-#include <cbuf_c.h>
+#include <cbuf_meta.h>
 #include <cbufp.h>
-#define cbid_to_meta_idx(cid) ((cid) << 1)
-#define meta_to_cbid_idx(mid) ((mid) >> 1)
+#define cbid_to_meta_idx(cid) ((cid) << 2)
+#define meta_to_cbid_idx(mid) ((mid) >> 2)
```

Comment:
BTW, the correct fix for this, further down the line, is to use the `ertrie`s in the `ppos` branch.  For now, this is fine, but we'll eventually want to replace the meta tree with that implementation (i.e. it is actually maintained and well-used).